gulpfile.js

```diff
@@ -76,6 +76,7 @@ declareExtension('amp-iframe', '0.1', false, 'NO_TYPE_CHECK');
 declareExtension('amp-image-lightbox', '0.1', true);
 declareExtension('amp-instagram', '0.1', false);
 declareExtension('amp-install-serviceworker', '0.1', false);
+declareExtension('amp-izlesene', '0.1', false, 'NO_TYPE_CHECK');
```

Comment:
remove `NO_TYPE_CHECK` and run `gulp check-types` to check types and fix issues accordingly.